src/sage/doctest/util.py

```diff
@@ -103,6 +103,95 @@ class Timer:
         {}
         sage: TestSuite(Timer()).run()
     """
+
+    def _pid_cpu_seconds(self, pid):
```

Comment:
Can you produce some doctests for this?

Reply:
These functions are basically random number generators, but I've done what I can.